client/utils/index.ts

```diff
@@ -212,9 +212,17 @@ export const createStore = (params = {}, app = planningApp) => {
     let _compose = compose;
 
     if (process.env.NODE_ENV !== 'production') {
-        // activate logs actions for non production instances.
-        // (this should always be the last middleware)
-        middlewares.push(createLogger());
+        const devtools = localStorage.getItem('devtools');
```

Comment:
What are you trying to achieve here? Will this remove the redux logs from the Browsers console?

Reply:
Does this then require the redux extension to work? I like to use the Browsers console, and it also has the ability to search so you can find what you need.

Could we use an env. variable to control this instead?

Reply:
It doesn't require a redux extension. It simply reads an array from local storage. Env var would mean that the config would need to be stored on the server and a network request would be required to get it, which seems like an overkill for such a simple thing.

Here's a line of code you can run in the console to enable it `localStorage.setItem('devtools', '["redux-logger"]')`

Reply:
In that case, could you add a section in this repos README.md file for redux-logs, and the instruction on enabling it please?